config/service.go

```diff
@@ -64,6 +71,23 @@ func (srvTL *ServiceTelemetryLogs) validate() error {
 	return nil
 }
 
+// ServiceTelemetryMetrics exposes the common Telemetry configuration for one component.
+type ServiceTelemetryMetrics struct {
```

Comment:
Please add a comment on all that this is experimental, so ensure we are not blamed if we change this.